Automattic/pocket-casts-android
# Convert the chapters fragment to Compose

## Description

This change converts the chapters fragment to Compose.

## Testing Instructions
1. Open the podcast Upgrade https://pca.st/upgrade
2. Play an episode
3. Tap on the mini player to open the full screen player
4. Tap on the Chapters tab
5. ✅ Verify the chapters are shown

## Screenshots or Screencast 

| Before Android | iOS | After Android |
| --- | --- | --- |
| ![Screenshot_20230906-135005](https://github.com/Automattic/pocket-casts-android/assets/308331/3e85497e-27ae-4f21-aa44-ce851f66963e) | ![IMG_5576AA3ADA93-1](https://github.com/Automattic/pocket-casts-android/assets/308331/b2608e58-42fa-42fc-a1d2-ca8bb5599cc0) | ![Screenshot_20230906_134911](https://github.com/Automattic/pocket-casts-android/assets/308331/f69efc49-ba3f-4b51-9d9f-6372460a2a8c) |
